package.json

```diff
@@ -44,11 +44,7 @@
     "compile:types": "typechain --target=truffle --outDir build/types/truffle-types \"build/contracts/*.json\"",
     "deploy:development": "npm run build && npm run migrate -- --network development",
     "deploy:kovan": "npm run build && npm run migrate -- --network kovan",
-    "compile:contracts": "truffle compile",
-    "compile:types": "typechain --target=truffle --outDir build/types/truffle-types \"build/contracts/*.json\"",
-    "build": "npm run compile:contracts && npm run compile:types",
     "pretest": "npm run build",
-    "test": "truffle test",
     "test:kovan": "truffle test --network kovan",
```

Comment:
I think also we will not need `test:kovan`, unit tests will be executed locally, right ?